### ISSUE TITLE
Overlay State in XT

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#4683]
Overlays are all shiny and new. Some questions though about things to test

- [x] Two MSEGS and flip between them with MSEG open - does the wave follow?
- [x] One MSEG one non-MSEG - does the button reset and the window close? When you return to the first MSEG is the button checked
- [x] MSEG and Formula - flip between does the flip or not? (And do we want it to?)
- [x] MSEG and Patch Store - if you combine these does everything work?
- [ ] MSEG and typeins - ditto

basically test a bunch of permutations and look for state or refresh errors now the ownership model has changed.

**Comment by mkruselj** [comment]
I think this one is good to close, @baconpaul.

The only thing I noticed that is different from 1.9 is that if you have a typein opened, and you open a MSEG, it doesn't close the typein (BUT typein is behind the MSEG editor, so that's ok in my book). After closing MSEG editor typein is still there and ready for input. I think that's fine, actually. Therefore, we could close it.

**Comment by baconpaul** [comment]
Yeah OK as long as it works that's fine, even if it is a bit different. Lets close this and if we find state problems in more extensive testing this fall add new issues.
